### PR TITLE
Added option in deliver to ignore language directory validation from …

### PR DESCRIPTION
**Pull request by joshdholtz**
…raising error

Fixes #9351
... again 😇 

### Problem
- Need to allow option to ignore language directory structure for invalid languages as not everyone will have same file structure
- Error message was also showing languages in all lowercase which wasn't correct

### Solution
- Added new option to ignore validation on metadata and screenshot directories
  - `ignore_language_directory_validation`
- Error message that gets raised will tell user about this new option
- Error message isn't all lowercase anymore

```sh
[!] Unsupported directory name(s) for screenshots/metadata in './fastlane/metadata': da-AAAAA
Valid directory names are: ["da", "de-DE", "el", "en-AU", "en-CA", "en-GB", "en-US", "es-ES", "es-MX", "fi", "fr-CA", "fr-FR", "id", "it", "ja", "ko", "ms", "nl-NL", "no", "pt-BR", "pt-PT", "ru", "sv", "th", "tr", "vi", "zh-Hans", "zh-Hant", "appleTV", "iMessage", "default"]

Enabled 'ignore_language_directory_validation' to prevent this validation from happening
```

### CODE REVIEW

File: deliver/lib/deliver/loader.rb [hunk 1]
```diff
@@ -33,9 +34,11 @@ def self.language_folders(root)
         File.directory?(path) && !allowed_directory_names.include?(normalized_path) && !EXCEPTION_DIRECTORIES.include?(normalized_path)
       end.sort
 
-      unless rejected_folders.empty?
+      if !ignore_validation && !rejected_folders.empty?
         rejected_folders = rejected_folders.map { |path| File.basename(path) }
-        UI.user_error! "Unsupport directory name(s) for screenshots/metadata: #{rejected_folders.join(', ')}\n\nValid directory names are: #{allowed_directory_names}"
+        UI.user_error! "Unsupported directory name(s) for screenshots/metadata in '#{root}': #{rejected_folders.join(', ')}" \
+                       "\nValid directory names are: #{allowed_directory_names_with_case}" \
+                       "\n\nEnabled 'ignore_language_directory_validation' to prevent this validation from happening"
```
Comment (mpirri) [inline deliver/lib/deliver/loader.rb]: "Enabled" -> "Enable"

**Review by mpirri — CHANGES_REQUESTED** [review]
Just that little text thing, but otherwise :100: !

File: deliver/lib/deliver/loader.rb (hunk 1 above)
Comment (joshdholtz) [inline deliver/lib/deliver/loader.rb]: I need a grammar checker 😔

**Comment by joshdholtz** [comment]
@mpirri Fixed! 💪 🔤

**Review by mpirri — APPROVED** [review]

**Pull request merged**